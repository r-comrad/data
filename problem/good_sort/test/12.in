66
5669785 9226513 8867576 3266960 8888592 1881206 8823901 2100115 1103542 5120302 2471677 8994849 145408 5243440 6626746 5042285 7520947 799958 6176875 9744655 4959351 7631969 5974095 3645728 5881310 2626465 500308 3135079 4155346 4546266 427985 7355285 8746203 4610736 7436988 2231989 1383598 7991122 8546814 25901 3827279 5782455 7147472 1475105 4759852 5133847 7395033 2611956 4487026 8193276 5704965 5745568 8337189 7917286 8421416 5806599 1526426 8697223 1932230 7863949 4136031 912581 6706693 5506288 1741557 3642332 