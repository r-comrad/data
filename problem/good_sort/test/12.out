5120302 500308 2100115 4610736 7520947 1881206 5881310 8421416 7991122 2611956 1741557 7631969 2471677 2231989 1932230 3642332 5782455 5042285 912581 8823901 7863949 5243440 1103542 5133847 8193276 1383598 8994849 9744655 5704965 5974095 145408 1475105 25901 3645728 7395033 4155346 5745568 3135079 7355285 8746203 8546814 9226513 1526426 4136031 6626746 2626465 3266960 4546266 6176875 7436988 5506288 6706693 5806599 4487026 8697223 7147472 3827279 8867576 427985 8337189 7917286 8888592 4959351 799958 4759852 5669785 